### DISCUSSION TITLE
Multiple WebView2 instances

### DISCUSSION THREAD

**Discussion by sachix1001** [tauri-apps/tauri#7904] [General]
I made Tauriapp using create-tauri-app with React/Typescript. When I build and installed the app, it runs with 6 WebView2 instances. I read running multiple WebView is by design but can someone confirm "6" of them is normal?

<img width="500" alt="Capture" src="https://github.com/tauri-apps/tauri/assets/45644289/3feacb15-d918-4c4e-a884-f630abb4fd66">

**Comment by FabianLars** [comment]
yep, that's normal. You can get a rough idea what the instances are when you open an empty window of a chromium based browser and open its internal task manager. Then you'll see one main process, one gpu process, one network process etc.

Edit: just found this page: https://learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/end-user-faq#what-memory-and-cpu-resources-does-webview2-use - so windows 11 even started listing the actual process names lately. (above that section it also explain what i said above)
